src/file/file.model.ts

```diff
@@ -13,13 +14,10 @@ export const fileSchema: mongoose.Schema = new mongoose.Schema(
       sparse: true,
       unique: true,
     },
-    displayName: {
+    name: {
```

Comment:
Should define unique index on `name` and `parent` fields.

Reply:
why not also owner?
there could be two files with the same name whose parent is null (root), but the owner is different

Reply:
and that makes another problem - what if the file is not owned by me, but is on the root and has the same name as the other file in root?

Reply:
You are absolutely right.

Reply:
You are right, add that too to the unique index.

Reply:
This isn’t possible, if a file is in a root folder it has to be under a certain bucket. Either it’s yours or not.